tests/microos/disk_boot.pm

```diff
@@ -20,7 +20,9 @@ sub run {
     # default timeout in grub2 is set to 10s
     # Sometimes, machines tend to stall when trying to match grub2
     # this leads to test failures because openQA does not assert grub2 properly
-    if ((is_aarch64 || is_sle_micro('>=6.0')) && !main_micro_alp::is_dvd()) {
+    # KEEP_GRUB_TIMEOUT=0 will force the grub needle to match, useful when booting
+    # pre-configured images with disabled timeout. See opensusebasetest::handle_grub
+    if ((is_aarch64 || is_sle_micro('>=6.0')) && get_var('KEEP_GRUB_TIMEOUT', '1') && !main_micro_alp::is_dvd()) {
```

Comment:
Can this be simplified? This is getting out of hand.

Reply:
how?

Reply:
```suggestion
    if (get_var('KEEP_GRUB_TIMEOUT', '1')) {
```

But this likely requires a LOT of finetuning on the jobs settings everywhere.

Let's keep it for now if it works, but we should avoid constructs like this one and get rid of them.

Reply:
Your decision. I find it more confusing than helpful and reusing existing variables for the sake of just reusing variables not a desirable goal.

Reply:
Ok. Let's just unblock the situation and think of a better way to handle it later on with more time and less pressure.